src/utils.cairo

```diff
@@ -133,3 +133,27 @@ pub fn double_sha256(a: u256, b: u256) -> u256 {
         low: x4.into() * TWO_POW_96 + x5.into() * TWO_POW_64 + x6.into() * TWO_POW_32 + x7.into(),
     }
 }
+
```

Comment:
I copied and pasted this in here because I couldn't find a way to bring them from tests/utils